lib/console.js

```diff
@@ -64,10 +64,9 @@ Console.prototype.time = function(label) {
 
 
 Console.prototype.timeEnd = function(label) {
-  var time = this._times.get(label);
-  if (!time) {
-    throw new Error('No such label: ' + label);
-  }
+  const time = this._times.get(label);
+  if (!time) return;
```

Comment:
Can you just replace the old `throw` with `return;`. It just needs to be a one line change.